lib/cadet/application.ex

```diff
@@ -19,6 +19,18 @@ defmodule Cadet.Application do
       worker(Guardian.DB.Token.SweeperServer, [])
     ]
 
+    # To supply command line args to phx.server, you must use the elixir/iex bin
+    #     $ elixir --erl "--updater" -S mix phx.server
+    #     $ iex --erl "--updater" -S mix phx.server
```

Comment:
Should be `cadet.server` me thinks

Reply:
It's aliased to `phx.server` at [`mix.exs`](https://github.com/source-academy/cadet/blob/896abe5e204b903745fb29f24fbd180140dc9e8c/mix.exs#L86)